src/Drupal/DrupalAutoloader.php

```diff
@@ -63,8 +63,8 @@ class DrupalAutoloader
     public function register(Container $container): void
     {
         $startPath = null;
-        $drupalParams = $container->hasParameter('drupal') ? $container->getParameter('drupal') : [
-          'drupal_root' => null,
+        $drupalParams = [
+          'drupal_root' => $container->getParameter('drupal'),
```

Comment:
drupal_root is a param under drupal.  but with the schema, I shouldn't have to worry about missing data, correct?

Reply:
Yes, it will always have this parameter.